Change - Update authentication options for iscsi::initiator type

blackknight36 opened the pull request:
Updated manifest to allow management of targets that do not use CHAP.

Commit:
Change - Update authentication options for iscsi::initiator type

Updated manifest to allow management of targets that do not use CHAP.

jflorian reviewed (changes requested):
@blackknight36 Please revise the README.

blackknight36 commented:
No problem.  I've updated the documentation for the parameters in the README file and added an example resource for defining a target with CHAP disabled.

jflorian reviewed (approved):
Nice, thanks!
